website/docs/prysm-usage/fee-recipient.md

```diff
@@ -0,0 +1,88 @@
+
+This document shows you how to configure Fee Recipient on your validator client instance. 
+
+:::tip Post-Merge
+This feature will activate after [The Merge](https://ethereum.org/en/upgrades/merge/).
+:::
+
+:::warning Known Release Bug
+Users are unable to set a default fee recipient through the validator client due to an implementation bug in release 2.0.7. This was fixed and merged into develop for the following release in pr [#10555](https://github.com/prysmaticlabs/prysm/pull/10555). Users can still set a default fee recipient through the beacon node if no fee recipient configurations are provided to the validator client.
+:::
+
+:::warning Incorrectly configured Fee Recipient settings could lead to a loss of earnings.
+
+To ensure that your Fee Recipient settings are correctly configured, ___________________.
+
+If you don't configure your Fee Recipient settings, the Fee Recipient address will default to the burn address (`0x000....0`). Be sure to specify a valid Ethereum wallet address (that you own) as the fee recipient. 
+
+:::
+
+## Configuring Fee Recipient on your validator client instance
+
+Fee Recipient is configured by either providing a Fee Recipient address, or the location of a JSON file that specifies your Fee Recipient address.
+
+The following configuration options are available:
+
+| Flag                        | Example                                                                                                                                                                                                                                              |
+|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `suggested-fee-recipient`   | Sets a default eth address for ALL validators.  Example: `--suggested-fee-recipient=0x046Fb65722E6b0000012BFEBf6177F1D2e9758D9`   Note: This setting overrides the two `config` options below. If you set this, the `config` options will be ignored |
+| `fee-recipient-config-file` | Sets the file location for the Fee Recipient JSON configuration.   Example: `--fee-recipient-config-file=./fee_recipient_config.json`                                                                                                                |
+| `fee-recipient-config-url`  | Sets a URL for a remote Fee Recipient JSON configuration.  Example: `--fee-recipient-config-url=http://example.com/api/getFeeRecipient`                                                                                                              |
+
+suggested fee recipient flag will override the configuration file.
+
+you can not use both the file and url flags at the same time.
+
+the validator client will send the fee recipient information at startup for all public keys that are active
```

Comment:
No need to go into implementation detail here. Let's keep it simple for the user